tools/urls-check

```diff
@@ -18,20 +18,15 @@
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
 
-import os
-from os.path import dirname, abspath
 import sys
 import time
 import argparse
 import subprocess
 import urllib
 from urllib.request import urlopen, Request
-
-sys.path.append(os.path.join(dirname(dirname(abspath(__file__))), "bots"))
+from parent import BASE_DIR
 import task
 
-sys.dont_write_bytecode = True
```

Comment:
This is fine, but should be in a separate commit.

Reply:
OK.